src/nsolid.cc

```diff
@@ -483,6 +483,28 @@ int CpuProfiler::get_cpu_profile_(SharedEnvInst envinst,
                                                    deleter);
 }
 
+int Snapshot::start_tracking_heap_objects_(SharedEnvInst envinst,
+                                           bool redacted,
+                                           bool trackAllocations,
+                                           uint64_t duration,
+                                           void* data,
+                                           snapshot_proxy_sig proxy,
+                                           internal::deleter_sig deleter) {
+  if (envinst == nullptr) return UV_ESRCH;
+
+  return NSolidHeapSnapshot::Inst()->StartTrackingHeapObjects(
+      envinst, redacted, trackAllocations, duration, data, proxy, deleter);
+}
+
+int Snapshot::stop_tracking_heap_objects_(SharedEnvInst envinst,
+                                          void* data,
+                                          snapshot_proxy_sig proxy,
+                                          internal::deleter_sig deleter) {
+  if (envinst == nullptr) return UV_ESRCH;
```

Comment:
Follow the syntax used in the rest of the code
```suggestion
  if (envinst == nullptr)
    return UV_ESRCH;
```

Reply:
Just letting you know, that is formatted by `make cpp-format`